File: 프로그래머스/unrated/181940. 문자열 곱하기/문자열 곱하기.java
class Solution {
    public String solution(String my_string, int k) {
        StringBuilder answer = new StringBuilder();
        for (int i = 0; i < k; i++) {
            answer.append(my_string);
        }
        return answer.toString();
    }
    public static void main(String[] args) {
        Solution sol = new Solution();
        String my_string = "string";
        int k = 3;
        String answer = sol.solution(my_string, k);
        System.out.println("answer = " + answer);
    }
}

File: JavaScript/프로그래머스/0/181940. 문자열 곱하기/문자열 곱하기.js
// Js에서 StringBuilder를 사용하지 않는 이유 : https://tech-monster.tistory.com/178

function solution(my_string, k) {
    let answer = '';
    for (let i = 0; i < k; i++){
        answer += my_string;
    }
    return answer;
}